core/controllers/certificate_assessment.py

```
@@ -400,25 +400,25 @@ class StartCertificateAssessmentHandler(
+        try:
+            attempt, questions = (
+                certificate_assessment_services.start_certificate_assessment_attempt(
+                    self.normalized_payload['certificate_id'], self.user_id
+                )
+            )
+        except (
+            certificate_assessment_services.CertificateAssessmentAttemptNotReadyException
+        ):
+            raise self.InvalidInputException(
+                'Sorry, this assessment isn\'t ready anymore! We\'ve alerted the creator, and in the meantime you can try a different assessment.'
+            )
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)